src/main/java/pages/cartPage/BaseTable.java

```diff
@@ -0,0 +1,58 @@
+package pages.cartPage;
+
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.BasePage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public class BaseTable extends BasePage {
+    protected String tableXpath;
+    public By tableLocator;
+    public By headerRow;
+
+    public BaseTable(String tableXpath) {
+        this.tableXpath = tableXpath;
+        this.headerRow = By.xpath(tableXpath + "/tbody/tr");
+        this.tableLocator = By.xpath(tableXpath);
+    }
+
+    public int getRowCount() {
+        WebElement table = locateElement(tableLocator);
```

Comment:
you can use var here and in the row below:)

Reply:
Changed method. Declaring a variable is no longer necessary 
```java
    public int getRowCount() {
        return basePage.locateElement(tableLocator)
                .findElements(By.tagName("tr"))
                .size();
    }
```